src/main/java/gov/nasa/pds/tools/util/TabulatedUtil.java

```diff
@@ -0,0 +1,63 @@
+package gov.nasa.pds.tools.util;
+
+import java.lang.StringBuffer;
+
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import javax.xml.transform.sax.SAXSource;
+
+import net.sf.saxon.tree.tiny.TinyNodeImpl;
+import net.sf.saxon.om.DocumentInfo;
+
+import gov.nasa.pds.tools.util.LabelParser;
+import gov.nasa.pds.tools.util.Utility;
+import gov.nasa.pds.tools.util.XMLExtractor;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class TabulatedUtil {
+  // Class to inspect a tabulated content from a table file.
+
+  private static final Logger LOG = LoggerFactory.getLogger(TabulatedUtil.class);
+  private static URL contextValue      = null;
+
+  public TabulatedUtil(URL contextValue) {
+      this.contextValue   = contextValue;
+  }
+
+  public URL getTarget() {
+    return(this.contextValue); 
+  }
+
+  public String getRecordDelimiter() {
+      // Get the value of 'record_delimiter' in a label.  The returned value can be null.
+      // <Table_Character>
+      //     <record_delimiter>Carriage-Return Line-Feed</record_delimiter>
+
+      String recordDelimiter = null;
+
+      List<TinyNodeImpl> xmlModels = new ArrayList<TinyNodeImpl>();
+      try {
+          SAXSource saxSource = new SAXSource(Utility.openConnection(getTarget()));
+          DocumentInfo docInfo = LabelParser.parse(saxSource); // Parses a label.
+          XMLExtractor extractor = new XMLExtractor(docInfo);
+          xmlModels = extractor.getNodesFromDoc("Product_Observational/File_Area_Observational/Table_Character");
```

Comment:
- [ ] 1. if we could use the `Xpaths` class to define these XPaths as variables that would great. see https://github.com/NASA-PDS/validate/blob/master/src/main/java/gov/nasa/pds/tools/validate/rule/pds4/TableDataContentValidationRule.java#L128 for an example of usage. hard coding in these classes can be really hard to track down in the future.

- [ ] 2. This does not just apply to `Table_Character`, this applies to all places where `record_delimiter` exists in a product. if possible, I would just generalize this to `//record_delimiter`. if that is not possible, then we will need to find all the objects this applies to in the [PDS4 IM Spec](https://pds.nasa.gov/datastandards/documents/im/v1/index_1G00.html)

Reply:
As it turns out, another class contains a function to fetch the delimiter already so there's no need to re-parse the label file.